src/move/conf/schema.py

```diff
@@ -170,6 +172,8 @@ class IdentifyAssociationsConfig(TaskConfig):
 class IdentifyAssociationsBayesConfig(IdentifyAssociationsConfig):
     """Configure the probabilistic approach to identify associations."""
 
+    multiprocess: bool = False  # Default value is False
```

Comment:
Would make more sense to add this as an attribute of the base class (`IdentifyAssociationsConfig`), since it will be shared among the child classes (Bayes, Ttest, Ks).

Reply:
true. I added it to the base class